src/internal/components/dropdown/styles.scss

```diff
@@ -115,3 +115,8 @@
 .stretch-trigger-height {
   block-size: 100%;
 }
+
+@include styles.with-direction('.dropdown:is(.dropdown-drop-left, .dropdown-drop-right)') {
```

Comment:
For instance, this selector looks okay for embedded string, but the icon one is too long

Reply:
See above comment!